### PR TITLE
Add AnimatedListDetail and LaunchCollection to liibrary

### PR DESCRIPTION
**Pull request by BottleRocket-Colin**
Moved AnimatedListDetail from Arch Demo and some supporting util functions.  Tested using MavenLocal and ArchDemo project.

### CODE REVIEW

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -23,28 +20,7 @@ object Config {
     // https://github.com/JLLeitschuh/ktlint-gradle/releases
     const val KTLINT_GRADLE_VERSION = "10.2.1"
 
-    // https://github.com/pinterest/ktlint/blob/master/CHANGELOG.md
```
Comment (jaredstockton) [inline buildSrc/src/main/kotlin/Dependencies.kt:26]: @BottleRocket-Colin I definitely understand why you've cleaned house on a majority of entries listed here (they aren't being used) and appreciate the effort!

That being said, I do think there is value to having a few of the buildscript/apply plugins (`detekt` and `gradle versions`) added back in some future PR. I'll add it to our todo list

**Review by jaredstockton — APPROVED** [review]
Nice work!

**Comment by jaredstockton** [comment]
@BottleRocket-Colin Just thought of something after initial approval. Go ahead and update the version to `0.3.0` mentioned under Usage in the README.md and this should be good to go!

**Review by jaredstockton — CHANGES_REQUESTED** [review]
See https://github.com/BottleRocketStudios/Android-LaunchPad-Compose/pull/2#issuecomment-1190639103

**Comment by BottleRocket-Colin** [comment]
implementation example in README.md updated to version 0.3.0

**Review by jaredstockton — APPROVED** [review]
Thanks for the quick turnaround!

**Pull request merged**